### PR TITLE
Fix anonymous grading in new DocViewer

### PR DESCRIPTION
**Pull request by vanwagonet**
Properly handle the case where annotation metadata is not returned.

refs: MBL-11927
affects: none
release note: none

### CODE REVIEW

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: None

[MBL-11927](https://instructure.atlassian.net/browse/MBL-11927)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Core** | 90.2% | 90.24% | -0.04%
**Student** | 94.56% | 94.56% | 0%
**Teacher** | 100% | 100% | 0%
**React Native** | 92.3% | 92.3% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Core/Core/Extensions/OperationQueueExtensions.swift [hunk 1]
```diff
@@ -22,7 +22,7 @@ extension OperationQueue {
     public func addOperationWithErrorHandling(_ group: OperationSet, sendErrorsTo view: ErrorViewController?) {
         addOperation(group, errorHandler: { error in
             if let error = error {
-                view?.showError(error)
+                DispatchQueue.main.async { view?.showError(error) }
```
Comment (bluwave) [inline Core/Core/Extensions/OperationQueueExtensions.swift]: Did you get an error with this?  ErrorViewControlller already dispatches errors on the main thread, although, it looks we don't dispatch `showError` on the main thread when it's a string.  🙁

Comment (vanwagonet) [reply Core/Core/Extensions/OperationQueueExtensions.swift]: I got a crash here.

Comment (vanwagonet) [reply Core/Core/Extensions/OperationQueueExtensions.swift]: The view has a showError method that stops animating a UIActivityIndicator.

**Pull request merged**